lib/active_admin/namespace_settings.rb

```diff
@@ -103,5 +103,17 @@ class NamespaceSettings < DynamicSettingsNode
 
     # Include association filters by default
     register :include_default_association_filters, true
+    register :maximum_association_filter_arity, 20
```

Comment:
I'm in favor of a higher value, say 256, that will preserve country selects.

Reply:
Would country selects be done with an association table? I'd assumed you would use the builtin rails select.

Reply:
If the countries live on a separate table, the yes, right? I tend to agree with @varyonic's opinion of setting a higher value here.